core/tokens.go

```diff
@@ -0,0 +1,91 @@
+package core
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/mr-tron/base58/base58"
+	"github.com/segmentio/ksuid"
+	"github.com/textileio/textile-go/crypto"
+	"github.com/textileio/textile-go/repo"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// CreateCafeToken creates a random developer access token, returns a base58 encoded version,
+// and stores a bcrypt hashed version for later comparison
+func (t *Textile) CreateCafeToken() (string, error) {
+	key, err := crypto.GenerateAESKey()
+	if err != nil {
+		return "", err
+	}
+
+	id := ksuid.New().String()
+	created := time.Now()
+	rawToken := key[:32]
```

Comment:
We only need 32 bytes really, so just grab those. Easier than re-implementing the random reader function we already have.